libs/wingsdk/src/util/util.ts

```diff
@@ -1,3 +1,6 @@
+import { SHA256 } from "crypto-js";
```

Comment:
why not use node.js's `crypto` library? (always prefer not to use a dependency if possible)